### DISCUSSION TITLE
measurement sequencing and data pipeline advice

### DISCUSSION THREAD

**Discussion by joamatab** [pymeasure/pymeasure#849] [Q&A]
What is the recommended intermediate format to define the testing procedure?

We were thinking about using YAML as an intermediate file format between [design](https://gdsfactory.github.io/gdsfactory/README.html) and pyMeasure validation

Also we are starting to build our [data pipeline ](https://github.com/gdsfactory/gdsfactory/issues/1227) and would be great to use the same format to go from raw to processed data

**Comment by bilderbuchi** [comment]
Our procedures are defined in python files, is that what you're asking?

I'll move this to the discussion forum, it's more appropriate there.

**Comment by joamatab** [comment]
How could we define an intermediate representation that can interface design tools with testing tools?
